src/jobservice/job/impl/context.go

```diff
@@ -51,9 +57,24 @@ func NewContext(sysCtx context.Context, adminClient client.Client) *Context {
 
 //Init ...
 func (c *Context) Init() error {
-	configs, err := c.adminClient.GetCfgs()
-	if err != nil {
-		return err
+	var (
+		counter = 0
+		err     error
+		configs map[string]interface{}
+	)
+
+	for counter == 0 || err != nil {
+		counter++
+		configs, err = c.adminClient.GetCfgs()
+		if err != nil {
+			logger.Errorf("Job context initialization error: %s\n", err.Error())
+			if counter < maxRetryTimes {
+				backoff := (int)(math.Pow(2, (float64)(counter))) + 2*counter + 5
+				time.Sleep(time.Duration(backoff) * time.Second)
```

Comment:
Could you add a log saying "retry in xxx seconds"?

Reply:
added